backend/app/tests/services/inquiry/test_create_inquiry.py

```diff
@@ -0,0 +1,54 @@
+from unittest.mock import Mock
+
+import pytest
+from pydantic import ValidationError
+
+from app.models.inquiry import Inquiry, InquiryCreate
+from app.services.inquiries import create_inquiry
+
+
+@pytest.fixture
+def mock_session() -> Mock:
+    return Mock()
+
+
+def test_should_create_with_valid_data(mock_session: Mock) -> None:
```

Comment:
This is an example of testing to implementation. Usually this is a bit of a smell -- is there a way to refactor so you're testing to outcome? i.e. what is the expected outcome from calling the FUT? Is there a way to verify that instead of verifying *how* the FUT is getting to that outcome?

Reply:
Again , I'm not sure waht you are asking.  THe service layer function is invoked.  We are therefore testing that the correct series of database calls are being invoked.

Reply:
This is the code being tested

```

def create_inquiry(*, session: Session, inquiry_in: InquiryCreate) -> Inquiry:
    db_inquiry = Inquiry.model_validate(inquiry_in)
    session.add(db_inquiry)
    session.commit()
    session.refresh(db_inquiry)
    return db_inquiry
```